lib/interface/helpers.rb

```diff
@@ -2,17 +2,40 @@
 
 module Interface
   module Helpers
-    # Errors raised here identify the class_name that is not implementing
-    # the method required by the Interface.
+    # Errors raised here identify the class_name that is not implementing or is
+    # implemented with the wrong arity required by the Interface.
     def must_implement(*args)
-      args.each do |method_name|
-        next if respond_to? method_name
+      args.each do |method|
+        method = method_info(method)
+        next if respond_to?(method[:name]) && expected_arity?(method)
         raise Interface::Error::NotImplementedError.new(
                 class_name: self.class.name,
-                method_name: method_name,
+                method_name: method[:name],
+                method_arity: method[:arity],
                 interface_name: self.class.ancestors.first
               )
       end
     end
+
+    private
+
+    # Build a hash with the method name and arity.
+    def method_info(method)
```

Comment:
Do we need this method? I thought `args.each do |method, arity| will yield a `nil` arity if the method is a symbol...